src/content/en/2020/markup.md

```diff
@@ -543,58 +608,63 @@ When we built our tests we didn’t check for the presence of images, but only l
 
 ### Button and input types
 
-There has been a lot of discussion on buttons lately and how often they are misused. We looked into this to present findings on some of the native HTML buttons:
+There has been a lot of [discussion](https://adrianroselli.com/2016/01/links-buttons-submits-and-divs-oh-hell.html) on buttons lately and how often they are misused. We looked into this to present findings on some of the native HTML buttons.
+
+{{ figure_markup(
+  caption="Percent of pages with button elements.",
+  content="60.56%",
+  classes="big-number",
+  sheets_gid="410549982",
+  sql_file="pages_markup_by_device.sql"
+) }}
 
 <figure markdown>
-| Elements | Occurrences | Page percentage |
+| Button types | Occurrences | Percentage |
 |---|---|---|
-| `<button type="button">…</button>` | 15,926,061 | 36.41% |
+| `<button type="button">` | 15,926,061 | 36.41% |
 | `<button>` without type | 11,838,110 | 32.43% |
-| `<button type="submit">…</button>` | 4,842,946 | 28.55% |
+| `<button type="submit">` | 4,842,946 | 28.55% |
 | `<input type="submit" value="…">` | 4,000,844 | 31.82% |
 | `<input type="button" value="…">` | 1,087,182 | 4.07% |
 | `<input type="image" src="…">` | 322,855 | 2.69% |
-| `<button type="reset">…</button>` | 41,735 | 0.49% |
+| `<button type="reset">` | 41,735 | 0.49% |
 
-<figcaption>{{ figure_link(caption="Top 7 button setups.", sheets_gid="410549982", sql_file="pages_markup_by_device.sql") }}</figcaption>
+<figcaption>{{ figure_link(caption="Adoption of button types.", sheets_gid="410549982", sql_file="pages_markup_by_device.sql") }}</figcaption>
 </figure>
 
-According to the numbers we gathered, about 60% of the pages contain a button element and more than half of them (32.43%) fail to specify a `type` attribute.
-
-<div class="note">
-  <p>The button element has a <a href="https://dev.w3.org/html5/spec-LC/the-button-element.html">default type of <code>submit</code></a>, meaning that the default behavior of a button is to submit the current form data. For extra clarity, consider specifying the button type in order to avoid unexpected situations.</p>
-  <p>The following table shows that a page in the 90th percentile has at least 13 native buttons, while the pages in the 10th and 25th percentiles contain no buttons at all:</p>
-</div>
+Our analysis shows that about 60% of pages contain a button element and more than half of the pages (32.43%) have at least one button that fails to specify a `type` attribute. Note that the button element has a [default type](https://dev.w3.org/html5/spec-LC/the-button-element.html) of `submit`, so the default behavior of buttons on these 32% of pages is to submit the current form data. To avoid possibly unexpected behavior like this, a best practice is to specify the type attribute.
 
-<!-- Really part of the note above, but kept outside for simplicity: -->
 <figure markdown>
-| Percentile | Buttons |
+| Percentile | Buttons per page |
 |---|---|
+| 10 | 0 |
```

Comment:
In my mind this is not adding precision—am I missing something or could it be that 25/0 is quite sufficient?